test/transforms/json-to-html-org.test.js

```diff
@@ -0,0 +1,17 @@
+const test = require('tape');
+const config = require('../../config');
+const buildJSONResponse = require('../../lib/jsonapi-response');
+const JSONAPIResponse = buildJSONResponse(require('../fixtures/example-get-response-organisation.json'), config);
+const buildHTMLData = require('../../lib/transforms/json-to-html-data');
+const HTMLData = buildHTMLData(JSONAPIResponse);
```

Comment:
This should be done within the test - since one thing you can test for is whether it throws!